src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php

```diff
@@ -83,7 +85,7 @@ public function catalogAction()
     /**
      * Controller responsible for displaying "Catalog Module Grid" section of Module management pages with ajax.
      *
-     * @AdminSecurity("is_granted(['read', 'create'], 'ADMINMODULESSF_')")
+     * @AdminSecurity("is_granted(['read', 'create', 'update', 'delete'], 'ADMINMODULESSF_')")
```

Comment:
Why do we ask for all privileges to reach the catalog content? Modules actions should are allowed accordingly.

Reply:
You can't delete something you can't see :D
This is how it's configure in many Symfony controller, just did the same thing :/